tutorials/12_Calibrating_CES_Parameters.md

```diff
@@ -89,149 +89,140 @@ To set up a CES calibration run, simply set module 29 `CES_parameters` to the
-the prices calculated as partial derivatives on the CES production function to
-be written to the file `pm_cesdata_price`, which might be helpful in finding a
-suitable value for `cm_CES_calibration_default_price`, but is generally not
-used.
+If your calibration depends on new input data, you need to update the configuration.
+Use the command `lastrev` to find the latest input data revisions on the cluster.
+If your choice is _rev1.23abc_, update the following line of the configuration file
+`./config/default.cfg` (without _rev_ and always with quotation marks):
+
+```R
+cfg$inputRevision <- "1.23abc"
+```
+
+The calibration can further be adjusted using the following switches:
+
+- `c_CES_calibration_iterations` (default = 10): Number of CES calibration
+  iterations (iterative adjustments of $\alpha_i$ parameters, see above).
+  There is no convergence criteria, so check the calibration quality manually
+  afterwards.  More iterations generally lead to better convergence but longer
+  runtime, as each iteration is an additional REMIND run: select an appropriate
+  slurm configuration to allow for longer runtime, usually more than 24 hours.
+- `c_CES_calibration_new_structure`: Switch to set a default price for the
+  primary production factors instead of deriving prices from a `.gdx` file.
+  If the structure of the CES tree has changed, this switch is necessary
+  otherwise the calibration aborts.
+  The switch also allows forcing specific prices in the first CES calibration
+  iteration. Do not use it unnecessarily.
+- `cm_CES_calibration_default_prices` (default = 0.01): Default price to be used
+  when `c_CES_calibration_new_structure` is set.  The price is not
+  differentiated by period, region, or production factor.  When the price is far
+  from the equilibrium prices, calibration takes longer or shows poorer convergence.
+  Errors can occur if prices are too high.
+- `c_CES_calibration_write_prices`: Writes the prices (calculated as partial 
+  derivatives on the CES production function) into the file `pm_cesdata_price`.
+  Can help finding a suitable value for `cm_CES_calibration_default_price`.
 
 
 ## Results
 
-The main output of the CES calibration are a .gdx file and an .inc file with all
-the CES parameters.  They are named based on the CES configuration, the
+The CES calibration outputs a `.gdx` file and an `.inc` file with all
+the CES parameters.  Their long name, for instance
+`indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-En_gdp_SSP2-Kap_debt_limit-Reg_62eff8f7`
+indicates the CES configuration, the
 GDP/population scenarios, the capital market module realisation and the region
-configuration (e.g. `indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SSP2-GDP_gdp_SSP2-En_gdp_SSP2-Kap_debt_limit-Reg_62eff8f7`).
+configuration (_62eff8f7_ for H12, _2b1450bc_ for EU21, defined in the
+[Wiki](https://gitlab.pik-potsdam.de/REMIND/wiki/-/wikis/Abbreviations-and-Regions)).
```

Comment:
Maybe we should move that also to this collection of tutorials as it is not publicly available?